tuna/rocmlir/load_job.py

```diff
@@ -69,19 +76,21 @@ def add_jobs(args, dbt):
                    .filter(dbt.session_table.valid == 1,
                            dbt.session_table.id == args.session_id)
     reasons = query.all()
+    if not reasons:
+      raise ValueError(f"No session matching ID {args.session_id}")
     if len(reasons) > 1:
       raise ValueError(f"More than one session matching ID {args.session_id}")
     reason = reasons[0].reason
 
     query = session.query(dbt.config_table.id)\
                    .filter(dbt.config_table.valid == 1)
```

Comment:
```suggestion
    configs = session.query(dbt.config_table.id)\
                   .filter(dbt.config_table.valid == 1).all()
```
removed the intermid var

Reply:
Done.